.github/workflows/windows-meson-msys2.yml

```diff
@@ -75,26 +242,51 @@ jobs:
       - name: Strip binary
         run: strip build/dosbox.exe
```

Comment:
Meson can strip for us with: `-Dstrip=true`
(hopefully it work on MSYS2)

Reply:
Nope, doesn't appear to. I shall investigate, because that would be the better option.

Reply:
(feel free to resolve this either way - if you keep this manual step or go w/ the meson-install approach).

Reply:
Meson only strips on `install`. Punting on this for now.